### PR TITLE
Expose getBuildConfig hook

### PR DESCRIPTION
**Pull request by robin-drexler**
This will allow to retrieve the build config for later use, in our case to build a jest-preset.

### CODE REVIEW

File: packages/webpack/README.md [hunk 1]
```diff
@@ -108,6 +108,10 @@ If you want to intialize a build of your application, you can do so using this u
 
 Using this utility mixin method, you can delete your `buildDir` and all of its contents. It returns a `Promise`.
 
+### `getBuildConfig()` ([override](https://github.com/untool/mixinable/blob/master/README.md#defineoverride))
```
Comment (ZauberNerd) [inline packages/webpack/README.md]: This method needs a `target` argument.

File: packages/webpack/mixin.core.js [hunk 2]
```diff
@@ -3,8 +3,8 @@ const { join } = require('path');
 
 const debug = require('debug')('untool:webpack:stats');
 const {
-  sync: { pipe, sequence },
-  async: { override },
+  sync: { pipe, sequence, override },
```
Comment (dmbch) [inline packages/webpack/mixin.core.js]: I would prefer renaming `override` to `overrideSync`, similar to https://github.com/untool/untool/blob/master/packages/express/mixin.core.js#L2-L3.

**Comment by dmbch** [comment]
@ZauberNerd this would close #84, would it not?

File: packages/webpack/mixin.core.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ const { join } = require('path');
 
 const debug = require('debug')('untool:webpack:stats');
 const {
-  sync: { pipe, sequence },
+  sync: { pipe, sequence, override: overrideSync },
```
Comment (dmbch) [inline packages/webpack/mixin.core.js]: Sorry if I was being unclear - I meant having both `overrideSync` and `overrideAsync`, as in..:

```javascript
const {
  sync: { sequence, pipe, override: overrideSync },
  async: { override: overrideAsync },
} = require('mixinable');
```

**Comment by ZauberNerd** [comment]
@dmbch yes, it would close #84 and #82 :P

But the question still stands whether it is a good idea to override this method in the `hops-postcss` package.
But I guess we can postpone that discussion until you're back in the office ;)

**Pull request merged**